packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -1004,5 +1004,5 @@
     }
 }
 
-@import (inline) "../../../../node_modules/@khanacademy/math-input/build/math-input.css";
+@import (inline) "../../../../node_modules/@khanacademy/math-input/dist/index.css";
 @import (inline) "../../../../node_modules/mathquill/build/mathquill.css";
```

Comment:
We can probably remove this since we're importing `math-input/build/math-input.css`

Reply:
Good call! I found a few other duplicate imports that could be trimmed out also.